docs/providers.rst

```diff
@@ -59,7 +59,7 @@ Please, check the docs provided in the following API reference for more details:
 :class:`~.PageObjectInputProvider`.
 
 
-Cache Suppport in Providers
+Cache Support in Providers
 ===========================
```

Comment:
```suggestion
==========================
```

Reply:
:)

Reply:
One line change with a new one line bug! Of course.